### ISSUE TITLE
Resolve duplicate entries

### ISSUE THREAD

**Issue by blainekasten** [babel/babel-upgrade#81]
I just ran the babel-upgrade command. However, the output resulted in one plugin being set twice, which when running build threw an error about their being a duplicate plugin.

Ideally, this babel-upgrade plugin would check for duplicate entries and remove.

here was my previous .babelrc, and .after

### Before

```
{
  "plugins": [
    "syntax-dynamic-import",
    "inline-react-svg",
    ["babel-plugin-styled-components", { "ssr": true, "displayName": false }]
  ],
  "presets": [
    [
      "env",
      {
        "targets": {
          "browsers": ["firefox >= 52"]
        },
        "modules": false
      }
    ],
    "stage-2",
    "flow",
    "react"
  ],
  "env": {
    "development": {
      "plugins": ["babel-plugin-idx"]
    },
    "production": {
      "plugins": ["babel-plugin-idx"]
    },
    "test": {
      "presets": [["env"], "react", "stage-2", "flow"],
      "plugins": ["transform-es2015-modules-commonjs"]
    }
  }
}
```

### After

```
{
  "plugins": [
    "@babel/plugin-syntax-dynamic-import",
    "inline-react-svg",
    ["babel-plugin-styled-components", { "ssr": true, "displayName": false }],
    "@babel/plugin-syntax-dynamic-import",
    "@babel/plugin-syntax-import-meta",
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-json-strings",
    [
      "@babel/plugin-proposal-decorators",
      {
        "legacy": true
      }
    ],
    "@babel/plugin-proposal-function-sent",
    "@babel/plugin-proposal-export-namespace-from",
    "@babel/plugin-proposal-numeric-separator",
    "@babel/plugin-proposal-throw-expressions"
  ],
  "presets": [
    [
      "@babel/preset-env",
      {
        "targets": {
          "browsers": ["firefox >= 52"]
        },
        "modules": false
      }
    ],
    "@babel/preset-flow",
    "@babel/preset-react"
  ],
  "env": {
    "development": {
      "plugins": ["babel-plugin-idx"]
    },
    "production": {
      "plugins": ["babel-plugin-idx"]
    },
    "test": {
      "presets": [
        ["@babel/preset-env"],
        "@babel/preset-react",
        "@babel/preset-flow"
      ],
      "plugins": [
        "@babel/plugin-transform-modules-commonjs",
        "@babel/plugin-syntax-dynamic-import",
        "@babel/plugin-syntax-import-meta",
        "@babel/plugin-proposal-class-properties",
        "@babel/plugin-proposal-json-strings",
        [
          "@babel/plugin-proposal-decorators",
          {
            "legacy": true
          }
        ],
        "@babel/plugin-proposal-function-sent",
        "@babel/plugin-proposal-export-namespace-from",
        "@babel/plugin-proposal-numeric-separator",
        "@babel/plugin-proposal-throw-expressions"
      ]
    }
  }
}
```

### Diff that came from babel-upgrade (truncated)

```
npx: installed 192 in 4.186s
🙌  Thanks for trying out https://github.com/babel/babel-upgrade!

Updating .babelrc config at .babelrc
Index: .babelrc
===================================================================
--- .babelrc	Before Upgrade
+++ .babelrc	After Upgrade
@@ -1,19 +1,33 @@
 {
   "plugins": [
-    "syntax-dynamic-import",
+    "@babel/plugin-syntax-dynamic-import",
     "inline-react-svg",
     [
       "babel-plugin-styled-components",
       {
         "ssr": true,
         "displayName": false
       }
-    ]
+    ],
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-json-strings",
+    [
+      "@babel/plugin-proposal-decorators",
+      {
+        "legacy": true
+      }
+    ],
+    "@babel/plugin-proposal-function-sent",
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-numeric-separator",
+    "@babel/plugin-proposal-throw-expressions"
```

### Duplicated "@babel/plugin-syntax-dynamic-import"

**Comment by ljharb** [comment]
Duplicate of #77, fixed by #80?

**Comment by blainekasten** [comment]
Classic noob not looking through the issues or pull requests... Thanks @ljharb
